api/s2n.h

```diff
@@ -178,6 +178,8 @@ extern struct s2n_cert_chain_and_key *s2n_cert_chain_and_key_new(void);
 S2N_API
 extern int s2n_cert_chain_and_key_load_pem(struct s2n_cert_chain_and_key *chain_and_key, const char *chain_pem, const char *private_key_pem);
 S2N_API
+extern int s2n_cert_chain_load_pem(struct s2n_cert_chain_and_key *chain_and_key, const char *chain_pem);
```

Comment:
I know it's not consistent with the other APIs, but this should take an explicit length instead of a null-terminated pointer. See #2752 and #2753.

Reply:
Sure I can take what is done in those PRs and move it over.